changelog.md

```diff
@@ -9,6 +9,7 @@ There's a small change in behavior for environments in which the launcher script
 
 ### Fixed issues
 - [MARATHON-8159](https://jira.mesosphere.com/browse/MARATHON-8159) Fixed a bug where `taskKillGracePeriodSeconds` parameter was updated incorrectly during a migration to 1.5.
+- [DCOS-38317](https://jira.mesosphere.com/browse/DCOS-38317) Fixed a bug where Marathon was unable to restart an app with failing health checks and a persistent volume.
```

Comment:
we should mention the other fixes as well in particular:
1. https://github.com/mesosphere/marathon/commit/4eca3453a44518fa33dc4e567a5bc7a6be88afce
2. https://github.com/mesosphere/marathon/commit/77d3c939bb827273b6e56ca202f33e6b14fca544
3. https://github.com/mesosphere/marathon/commit/0701158e1cca6496fa7782eb21fd309563fa9334
4. https://github.com/mesosphere/marathon/commit/1256177099ed7b966f2be55d604d2ca09dede07a

Reply:
expanding the file in review... it looks like some of these are covered...    It would be good to add the ENV changes... but otherwise looks good.